From 819941f26c9aac2a5f06f29f8a8a9f202b347173 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Wed, 4 Dec 2024 09:46:09 +0100
Subject: [PATCH 19/95] usb: usbh-stm32: fix aarch32 build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix aarch32 build error:

error: implicit declaration of function ‘FIELD_PREP’
[-Werror=implicit-function-declaration]
 FIELD_PREP(SYSCFG_USBHARCR_OFFSET_AREN_MASK,

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/linux.git 52878edfeb355bc243445c0747ef4ede7b9dfb6a usb: usbh-stm32: fix aarch32 build]
Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I8c725bf231b931e301d6cd047fab7da3c4000d70
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/422530
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 drivers/usb/host/usbh-stm32.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/host/usbh-stm32.c b/drivers/usb/host/usbh-stm32.c
index 2c4ed0d271aea..f4052b69dc0ac 100644
--- a/drivers/usb/host/usbh-stm32.c
+++ b/drivers/usb/host/usbh-stm32.c
@@ -7,6 +7,7 @@
  * Author: Pankaj Dev <pankaj.dev@st.com>
  */
 
+#include <linux/bitfield.h>
 #include <linux/usb.h>
 #include <linux/pinctrl/consumer.h>
 #include <linux/mfd/syscon.h>
-- 
2.51.0

